smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/StructuredMemberWriter.java

```diff
@@ -65,6 +70,140 @@ void writeMembers(TypeScriptWriter writer, Shape shape) {
         }
     }
 
+    /**
+     * Recursively writes filterSensitiveLog for CollectionShape.
+     */
+    void writeCollectionFilterSensitiveLog(TypeScriptWriter writer, MemberShape collectionMember) {
+        Shape memberShape = model.expectShape(collectionMember.getTarget());
```

Comment:
It doesn't look like the cases where `List`, `Set`, or `Map` (below) members can be marked with the `@sensitive` trait.

Reply:
Good catch! This was an edge case which was missed

It was fixed in the following commits:
* Add writeStructureFilterSensitiveLog https://github.com/awslabs/smithy-typescript/pull/170/commits/cf462586a2e1fa38b0be84ecb1035f391961f7b8
* Edge case for collection with sensitive trait https://github.com/awslabs/smithy-typescript/pull/170/commits/7503ee764e0201d9de09d9d89154cfb6085b7c83
* Edge case for map with Sensitive trait https://github.com/awslabs/smithy-typescript/pull/170/commits/28d478bb080ff513f61b24ec5f8027cdd067bc54